tests/enmeshed-runtime.yml

```diff
@@ -1,7 +1,6 @@
 ---
 runner: command
-dir: enmeshed-runtime
-args: ["sh", "-c", "npm i && CONNECTION_STRING=mongodb://localhost:27017 npx jest -i"]
+args: [../enmeshed-runtime.sh]
```

Comment:
@b1ron I think we should document/promote that approach

Reply:
Nice, a much cleaner approach.